test/integration/targets/ansible/runme.sh

```diff
@@ -7,3 +7,12 @@ ansible --help
 
 ansible testhost -i ../../inventory -m ping  "$@"
 ansible testhost -i ../../inventory -m setup "$@"
+
+set -o pipefail
```

Comment:
Go ahead and set this at the start of the script.

Reply:
+1 to @mattclay